### ISSUE TITLE
Initialize broker topics and registry schemas

### ISSUE THREAD

**Issue by slominskir** [JeffersonLab/jaws-admin-gui#32]
Currently we rely on the jaws-libp container to initialize Kafka topics and also initialize the Schema Registry.   This should be done via this app (delegating to jaws-libj) and then the jaws-libp container would not be needed when the admin GUI app is in use.   Probably the jaws-libj command line needs to be shored up as well if it is to be a drop in replacement (and  jaws-libj CLI scripts also must be installed into the admin GUI container).

**Comment by slominskir** [comment]
Getting Kafka configured quickly is in the critical path and the Wildfly app server boots relatively slowly so consider simply moving scripts to admin container.
